src/main/java/it/smartcommunitylab/aac/webauthn/service/WebAuthnUserAccountService.java

```diff
@@ -0,0 +1,167 @@
+package it.smartcommunitylab.aac.webauthn.service;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import com.yubico.webauthn.data.ByteArray;
```

Comment:
remove usage of this class outside webauthn models and specific services.

Reply:
Addressed in 1087830eaa48c4faa46627f1b7c848eab27c8805